index.html

```diff
@@ -153,7 +153,7 @@ <h2>Contact me</h2>
         that you need built or a project that needs coding. I’d love to help
         with it
       </p>
-      <form action="https://formspree.io/f/mqkwbzqd" method="POST">
+      <form action="https://formspree.io/f/mqkwbzqd" method="POST" id="contact-form">
         <input
           type="text"
           name="user-name"
```

Comment:
- [ ] Good job implementing the form validation, it works as expected! However, there are a few tweaks needed for you to make and then you'll be good to go.
- Remember that you need to stick to the Figma design template as much as possible, including Fonts, typographies, colours, images, positioning and spacings. You are currently missing the background image decoration in the headline, about and contact sections. Kindly add these and fix the stylings that need to be fixed.

_See images below for more context._

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/123400462-a3fd6e00-d55a-11eb-8cf6-be399c6a0c8d.png) | ![image](https://user-images.githubusercontent.com/49621279/123400400-934cf800-d55a-11eb-89a4-21dd215769cd.png)
![image](https://user-images.githubusercontent.com/49621279/123400881-140bf400-d55b-11eb-8726-05228f4fad24.png) | ![image](https://user-images.githubusercontent.com/49621279/123400729-ecb52700-d55a-11eb-8e8e-dae339f1b55c.png)
![image](https://user-images.githubusercontent.com/49621279/123400938-24bc6a00-d55b-11eb-81dc-c4ffe25514f9.png) | ![image](https://user-images.githubusercontent.com/49621279/123400981-30a82c00-d55b-11eb-97a0-1a60051c5a86.png)